pkg/data/account.go

```diff
@@ -0,0 +1,132 @@
+package data
+
+import (
+	gErrors "errors"
+	"fmt"
+	"strconv"
+
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/Optum/dce/pkg/model"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+// Account - Data Layer Struct
+type Account struct {
+	DynamoDB       dynamodbiface.DynamoDBAPI
+	TableName      string `env:"ACCOUNT_DB"`
+	ConsistentRead bool   `env:"USE_CONSISTENT_READS" envDefault:"false"`
+}
+
+// WriteAccount the Account record in DynamoDB
+func (a *Account) WriteAccount(account *model.Account, lastModifiedOn *int64) error {
```

Comment:
lastModifiedOn here is the previous lastModifiedOn.  not sure that is clear enough.

Reply:
Maybe make explicit? Like `prevLastModifiedOn`?
Also, could elaborate in method comments

Reply:
Switched it.